lib/tape/ansible_runner.rb

```diff
@@ -89,9 +89,11 @@ def exec_ansible(playbook, args)
 
   def enforce_roles_path!
     Dir.mkdir('.tape') unless Dir.exists?('.tape')
-    File.open('.tape/ansible.cfg', 'w') do |f|
+    File.open("#{local_dir}/.tape/ansible.cfg", 'w') do |f|
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.